docs/basics/queries.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Queries
+order: 1
+---
+
+# Queries
+
+Let's get to querying our data! This page will teach us how we can retrieve our data declaratively
+from our GraphQL API with the help of `urql`.
+
+## React & Preact
+
+This guide covers how to query data with React and Preact, which share almost the same API.
+
+Both libraries offer a `useQuery` hook and a `Query` component. The latter accepts the same
+parameters but we won't cover it in this guide. [Look it up in the API docs if you prefer
+render-props components.](../api/urql.md#components)
+
+### Run a first query
+
+For the following examples imagine we are querying data from a GraphQL API that contains todo items
+Let's dive right into an example!
+
+```jsx
+import { useQuery } from 'urql';
+
+const TodosQuery = `
+  query {
+    todos {
+      id
+      title
+    }
+  }
+`;
+
+const Todos = () => {
+  const [result, reexecuteQuery] = useQuery({
+    query: TodosQuery,
+  });
+
+  const { data, fetching, error } = result;
+
+  if (fetching) {
+    return <p>Loading...</p>;
+  } else if (error) {
```

Comment:
Need to get rid of this unnecessary else - there's quite a few in the codebase also

```suggestion
  }
  
  if (error) {
```

Reply:
I’m curious, do you think it’s an issue with readability? I assumed since it’s equivalent both is fine 🤗

Reply:
Yes I think adding unnecessary conditional logic an issue with readability